src/footer/footer.html

```diff
@@ -53,6 +53,7 @@
 
     <div class="bottom">
       <div class="copyright"><img src="/cc.svg">2021</div>
```

Comment:
I don't see 2021 in the screen shot.  I also don't see the copyright icon.  Aren't you supposed to get rid of those?

Reply:
Sorry, I don't understand. What screenshot? The 2021 icon and copyright icon are still there, I did not get rid of them.

Reply:
The screenshot in the notion card that shows what this PR is supposed to be doing